gen/dcos-config.yaml

```diff
@@ -96,9 +96,27 @@ package:
         "name": "dummy",
         "type": "bridge"
       }
-  - path: /etc/dcos/network/overlay
+  - path: /etc/dcos/network/overlay-master
     content: |
       {{ dcos_overlay_network }}
+  - path: /etc/dcos/network/overlay-agent-master
+    content: |
+      {
+        "master": "zk://leader.mesos:2181/mesos",
```

Comment:
Only the leading Master can allocate the subnet, vtep IP and MAC, so the Agent (even when running on the Master) will always talk to the leading master.

Ofcourse, in the process of implementing the replicated log for the modules, so we don't currently support a multi master setup. Will do by end of this week.